content/docs/command-reference/fetch.md

```diff
@@ -92,7 +95,7 @@ or `-T` options are used).
 
 - `-j <number>`, `--jobs <number>` - number of threads to run simultaneously to
   handle the downloading of files from the remote. The default value is
-  `4 * cpu_count()`. For SSH remotes, the default is just `4`. Using more jobs
+  `4 \* cpu_count()`. For SSH remotes, the default is just `4`. Using more jobs
```

Comment:
why is it needed?

Reply:
Linter added I think... Will check again ⏳